website/blog/2020-08-31-Hydra_1.0.md

```diff
@@ -0,0 +1,431 @@
+tags: [Hydra, Release]
+---
+
+After many months and a lot of hard work by multiple people, Hydra 1.0 is finally out!
+
+Before jumping into the awesome new features, here are some statistics to illustrate how big this release is:
+
+The lines of code, examples and documentation in Hydra 1.0 more than tripled since Hydra 0.11.
+The number of unit tests more than doubled and is now 1577.
+Despite these big changes, this is still the Hydra you love - only bigger, stronger and with more heads.
+Let's get into it.
+
+<details><summary>Hydra 0.11 source code statistics</summary>
+
+```text title="$ cloc ~/hydra-0.11.tar.bz2 --include-lang=Python,Markdown,YAML"
+-------------------------------------------------------------------------------
+Language                     files          blank        comment           code
+-------------------------------------------------------------------------------
+Python                          82            898            605           4634
+Markdown                        42            381              0           1532
+YAML                            93             59            108            683
+-------------------------------------------------------------------------------
+SUM:                           217           1338            713           6849
+-------------------------------------------------------------------------------
+```
+
+</details><details>
+
+<summary>Hydra 1.0 source code statistics</summary>
+
+```text title="$ cloc ~/hydra-master.tar.bz2 --include-lang=Python,Markdown,YAML"
+-------------------------------------------------------------------------------
+Language                     files          blank        comment           code
+-------------------------------------------------------------------------------
+Python                         205           3011           1640          16565
+Markdown                       115           1414              1           5698
+YAML                           168             99            219            885
+-------------------------------------------------------------------------------
+SUM:                           488           4524           1860          23148
+-------------------------------------------------------------------------------
+```
+
+</details>
+
+<br/>
+
+### Breaking changes
+For most people, migrating from 0.11 to 1.0 will be smooth.
+However, you may have issues if you are relying on Python 2 support or have used internal APIs.
+- Hydra 1.0 drops support for [Python 2](https://www.python.org/doc/sunset-python-2/).
+- If you are relying on internal APIs Hydra 1.0 will likely break your code. Maintaining backward compatibility for internal APIs is not a goal.
+- Hydra 0.11 Configs pickled and stored will not unpickle with Hydra 1.0 due to internal changes in OmegaConf.
+
+There are multiple new deprecation warnings, each with a link to mini migration guide.
```

Comment:
link to a mini migration guide.

Reply:
There are 4 different pages. and no way to link to all of them.

Reply:
oops I meant "link to mini migration guide" -> "link to a mini migration guide"